examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,233 @@
+# Walkthrough of Time Log
+
+This is an example of using Kanister to protect a simple application called Time Log. This application is contrived but useful for demonstrating Kanister's features. Every second it appends the current time to a log file.
+
+Note: This application is used in the Kanister tutorial, for a detailed walkthrough click [here](https://docs.kanister.io/tutorial.html#tutorial).
```

Comment:
what are we trying to convey by this sentence, seems incomplete to me.

for detailed walk through of what?

Reply:
if you open the link, you'll find the example of time-log in kanister tutorial. And in that we're doing a detailed walkthrough of kanister using time-log application